tasks/main.yml

```diff
@@ -236,15 +236,15 @@
     - name: Set the bootstrap specific tftp file
       template:
         src: ../templates/pxe-bootstrap.j2
-        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ bootstrap.macaddr | regex_replace (':', '-')}}"
+        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ bootstrap.macaddr | lower | regex_replace (':', '-')}}"
         mode: 0555
       notify:
         - restart tftp
 
     - name: Set the master specific tftp files
       template:
         src: ../templates/pxe-master.j2
-        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | regex_replace (':', '-')}}"
+        dest: "/var/lib/tftpboot/pxelinux.cfg/01-{{ item.macaddr | lower | regex_replace (':', '-')}}"
```

Comment:
Doesn't the `lower` in the `with_items` section satisfy the requirement?

Reply:
You are right. Fixed.